packages/dds/tree/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+    "extends": [
+        "@fluidframework/eslint-config-fluid/eslint7"
```

Comment:
```suggestion
        "@fluidframework/eslint-config-fluid"
```

Reply:
I don't know that we've made a decision to not use namespaces, but I do think we should use them explicitly with intention. If they're doing something useful here, then go ahead and disable that rule.

Reply:
I do use them extensively, though I'm not opposed to doing things a different way if someone wants to suggest something else. I've disabled the rule for now.